community/modules/scheduler/schedmd-slurm-gcp-v6-controller/modules/slurm_files/scripts/slurmsync.py

```diff
@@ -327,20 +339,11 @@ def ignore_err(e) -> bool:
 
 def sync_placement_groups():
     """Delete placement policies that are for jobs that have completed/terminated"""
-    keep_states = frozenset(
-        [
-            "RUNNING",
-            "CONFIGURING",
-            "STOPPED",
-            "SUSPENDED",
-            "COMPLETING",
-        ]
-    )
 
     keep_jobs = {
-        str(job["job_id"])
-        for job in json.loads(run(f"{lookup().scontrol} show jobs --json").stdout)["jobs"]
-        if "job_state" in job and set(job["job_state"]) & keep_states
+        str(job.id)
+        for job in lookup().get_jobs()
+        if job.job_state and job.job_state in KeepStates
```

Comment:
`if job.job_state and job.job_state in KeepStates` -> `if job.job_state in KeepStates`